ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml

```diff
@@ -46,6 +48,27 @@ spec:
           --pyxis-url $(params.pyxis_url) \
           --cert-project-id $(params.cert_project_id)
 
+        ORGANIZATION=$(cat config.yaml | yq -r '.organization')
+        OPERATOR_DISTRIBUTION=$(cat cert_project | jq '.operator_distribution'| tr -d '"')
+        case "$ORGANIZATION" in
+          "certified-operators")
+            if [ "$OPERATOR_DISTRIBUTION" != "connect" ]; then
+              echo "The operator distribution $OPERATOR_DISTRIBUTION does not belong to $ORGANIZATION organization"
```

Comment:
Please change this message (and the similar one below) to:
```
This project is not permitted to distribute to the $ORGANIZATION catalog.
```

Reply:
Updated!